packages/xstate-graph/test/graph.test.ts

```diff
@@ -347,6 +347,39 @@ describe('@xstate/graph', () => {
 
       expect(adj).toHaveProperty('"full" | {"count":5}');
     });
+
+    it('should get events via function', () => {
```

Comment:
Something weird has happened here - this branch contains changes from the other PR: https://github.com/davidkpiano/xstate/pull/1663